input/pagecontent/4-3-security-considerations.md

```diff
@@ -29,67 +29,6 @@ FHIRcast builds on SMART by introducing a new [syntax for standard OAuth 2.0 sco
 * [Argonaut Risk Assessment report](http://argonautwiki.hl7.org/images/e/ed/%282015May26%29RiskAssessment_ReportV1.pdf).

```

Comment:
Above in the Actors section, the Resource server actor can be removed, correct?

Reply:
I don't think the RS actor was added because of webhooks, rather with the idea that a subscribing client would use their access token to also access the driving app's FHIR server.